packages/components/Molecules/DataTable/__test__/DataTable.test.js

```diff
@@ -0,0 +1,66 @@
+import { shallowMount } from '@vue/test-utils'
+import DataTable from '../DataTable'
+
+describe('DataTable', () => {
+  it('assigns props accordingly', () => {
+    const wrapper = shallowMount(DataTable, {
+      propsData: {
+        caption: 'Data table demo title'
+      }
+    })
+
+    expect(wrapper.vm.caption).toEqual('Data table demo title')
+    expect(wrapper.vm.isRowOriented).toBe(true)
+    expect(wrapper.vm.items).toEqual({})
+
+    wrapper.setProps({ items: {} })
+    expect(wrapper.vm.items).toEqual({})
+  })
+
+  it('assigns header and items for row oriented layout', () => {
+    const wrapper = shallowMount(DataTable, {
+      propsData: {
+        caption: 'Data table demo title',
+        items: {
+          0: ['Band', 'Singer', 'Inception', 'Label'],
+          1: ['Napalm Death', 'Barney Greenway', '1981', 'Century Media'],
+          2: ['Carcass', 'Jeff Walker', '1985', 'Earache'],
+          3: ['Extreme Noise Terror', 'Dean Jones', '1985', 'Candlelight'],
+          4: ['Discordance Axis', 'Jon Chang', '1992', 'Hydrahead'],
+          caption: 'table caption'
```

Comment:
I guess this answers my above question, but I tested in the storybook control panel doesn't seem change anything?

Reply:
😢  Hmm I might need to specify the control argType to handle object type better. Will update.

Reply:
Upgraded storybook to 6.2.7 which handles addon control of array of object type data. It works but it doesn't update automatically (you have to switch to docs tab so it works). I think this is still current issue similar to this https://github.com/storybookjs/storybook/issues/14508